src/point.cpp

```diff
@@ -37,6 +38,16 @@ point point::rotate( int turns, const point &dim ) const
     return *this;
 }
 
+float point::distance( const point &rhs ) const
+{
+    return sqrt( static_cast<float>( pow( x - rhs.x, 2 ) + pow( y - rhs.y, 2 ) ) );
```

Comment:
This line also needs to be updated.

Reply:
Ah, ok... thought I have to remove the `std::`, but this gives the above LGTM alerts. So I guess including `cmath` and keeping `std::` is the way to go....

Reply:
Done. No respective CI fails anymore.